src/main/resources/application.properties

```diff
@@ -6,5 +6,5 @@ spring.datasource.password=${DB_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 server.port=8080
 server.servlet.context-path=/api
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
why do you need that? use liquibase for creating your db schema and revert this change back to `validate`